indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -214,6 +215,7 @@ def _process(self):
             db.session.rollback()
             raise ExpectedError(unicode(e))
 
+        signals.rb.booking_created.send(resv)
```

Comment:
call this in `Reservation.create_from_data`? (right before `notify_creation`, also flush before triggering the signal just in case)